lib/common-types.ts

```diff
@@ -838,6 +851,30 @@ export type MRActions = {
     get: { params: GetSpaceParams & { previewApiKeyId: string }; return: PreviewApiKeyProps }
     getMany: { params: GetSpaceParams & QueryParams; return: CollectionProp<PreviewApiKeyProps> }
   }
+  Release: {
+    get: {
+      params: GetReleaseParams
+      return: ReleaseProps
+    }
+    query: {
+      params: GetSpaceEnvironmentParams & { query?: ReleaseQueryOptions }
+      return: CollectionProp<ReleaseProps>
+    }
+    create: {
+      params: GetSpaceEnvironmentParams
+      payload: ReleasePayload
+      return: ReleaseProps
+    }
+    update: {
+      params: GetReleaseParams & { version: number }
+      payload: ReleasePayload
+      return: ReleaseProps
+    }
+    delete: {
+      params: GetReleaseParams
+      return: null
```

Comment:
Should this be null, void, or undefined?

Reply:
(moved it to void)